simulators/README.md

```diff
@@ -9,14 +9,54 @@ There are extra features in [extras](extras):
 - the **import data** to upload Measurements or/and Alarms from data json file to OEE profiles.
 
 ## Simulator Microservice
+Detailed feature list:
+- configuration in JSON, no need to write code.
+- automatically creates devices and sends data.
+- identifies devices using a configurable `externalId`.
+- devices can be disabled to not send any events and measurements.
+- Written in python which can be modified easily for further development.
+- Simulators work based on status and shift that it is assigned to.
```

Comment:
please mention that the shift is linked via the locationId

Reply:
It is already mentioned  in the explanation of simulator
```suggestion
- Simulators act according to given Shiftplans
  - Simulators are linked to shiftplans via locationId
```